src/components/Icons/Icon.tsx

```diff
@@ -69,6 +82,7 @@ export default function Icon(props: React.PropsWithChildren<IconProps>) {
     className,
     desc,
     iconRotation,
+    color = "ui-black",
```

Comment:
Use `IconColors.ui_black` instead.

Reply:
Oiy, sorry. I got an email you added me as a reviewer but now I'm actually not seeing that message on this page. I'll refrain from commenting!